packages/v4/patternfly-docs/pages/componentGallery.css

```diff
@@ -0,0 +1,48 @@
+/* Toolbar styles */
```

Comment:
It seems like some of this could be mitigated by being able to remove padding from `#main-content` on the workspace, and probably putting this element higher up in the DOM so it isn't a child of the mdx-content elements that manage the example/sidebar layout. Probably move up and replace the `.ws-mdx-child-template` element? Then we wouldn't have to account for the `--pf-c-page__main-section--Padding[...]` vars

Reply:
Agreed this would have made this code much easier, but requires structural changes to how `documentation-framework` scaffolds docs based on wrapper elements & inserted parsed markdown docs.  This is outside of the scope of this issue, but may be a future enhancement we look into and evaluate other repercussions especially now that this code is being used outside of just PF docs.